.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: "1.52.0"
+          toolchain: stable
```

Comment:
Wait but why are you changing this?  I think pinning on 1.52 is useful for e.g. https://github.com/EmbarkStudios/cfg-expr/pull/31

Reply:
I could revert it, but I will have to re add the deprecated lints. Would it be more efficient to test compilation to rust 1.52 than forcing Clippy to be at 1.52 since Clippy as an option for msrv?